app/models/JsonFormats.scala

```diff
@@ -0,0 +1,22 @@
+package models
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDateTime, LocalTime, ZoneOffset}
+import java.util.Date
+
+
+object JsonFormats {
+
+  import reactivemongo.bson.{BSONObjectID, BSONDateTime}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)